scale/scale/settings.py

```diff
@@ -245,11 +248,18 @@
         'stream': sys.stdout
     },
     'console-err': {
-        'level': 'WARNING',
+        'level': 'DEBUG',
         'class': 'logging.StreamHandler',
         'formatter': 'standard',
         'stream': sys.stderr
     },
+    'console-debug': {
```

Comment:
I'd call this `console-out` or `console-stdout` and change `console-err` to `console-stderr`. Pick your consistent poison. :-) `console-debug` just doesn't indicate its target output to me.